shared_model/interfaces/common_objects/types.hpp

```diff
@@ -18,12 +18,12 @@
 #ifndef IROHA_SHARED_MODEL_TYPES_HPP
 #define IROHA_SHARED_MODEL_TYPES_HPP
 
-#include <boost/range/any_range.hpp>
 #include <cstdint>
 #include <set>
 #include <string>
 #include <vector>
 
+#include <boost/range/any_range.hpp>
```

Comment:
Not the right place for this include

Reply:
We have resolved that it is OK according to google codestyle.